scripts/rs-wnfs.sh

```diff
@@ -135,12 +135,11 @@ build_accumulator() {
 build_wasm() {
     display_header "💿 | BUILDING WNFS-WASM PROJECT | 💿"
     cd $script_dir/../wnfs-wasm
-    WASM_BINDGEN_WEAKREF=1 wasm-pack build --target web
+    WASM_BINDGEN_WEAKREF=1 npm run build
```

Comment:
We're now running `wasm-bindgen` ourselves, so we can get rid of this env variable and instead pass `wasm-bindgen --weak-refs` directly.
```suggestion
    npm run build
```